vmm/src/default_syscalls/x86_64.rs

```diff
@@ -14,7 +14,6 @@ pub fn default_filter() -> Result<SeccompFilter, Error> {
         vec![
             #[cfg(target_env = "musl")]
             allow_syscall(libc::SYS_accept),
```

Comment:
Is `SYS_accept` also used or have all moved to `SYS_accept4`?

Reply:
This is worth checking. It might be that they're all using SYS_accept4 instead of SYS_accept. I'll let you know.

Reply:
musl also switched to accept4, so I deleted the whitelisted accept.